apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -357,6 +362,9 @@ export default class CoreLibrary {
    */
   addSprite(opts) {
     opts = opts || {};
+    if (this.getNumberOfSprites() >= MAX_NUM_SPRITES) {
```

Comment:
We discussed moving this comparison to a helper function and using it here as well as in spriteCommands.

Reply:
Since this is only adding a single sprite, I think that this comparison suffices. But there are two other comparisons where I've updated the code with the helper function - thanks!

Reply:
Given the purpose of the new helper function, I agree.